docker-compose.yml

```diff
@@ -3,8 +3,8 @@ version: '3'
 services:
 
   nginx:
-    image: nginx:1.17.10
-    container_name: dsp_nginx
+    image: nginx:1.21.6
+    container_name: nginx
```

Comment:
I had the name as dsp_nginx to avoid conflicts with the nginx container built with hydroshare.  It's not a big deal though.

Reply:
They won't be running both on same VM or the same docker network. Container names are just local names on the docker network. I was just looking for some consistency. The database container is called "database." I call the nginx container "nginx" on very VM I run it on. It was little awkward to have to remember "dsp_nginx" when looking at logs, etc.. I was just taking care of a nit for myself.